### ISSUE TITLE
refetchOnFocus rate limiting?

### ISSUE THREAD

**Issue by pserrer1** [reduxjs/redux-toolkit#2613]
Hi,

is there a way to enable some sort of rate limiting for refetch on focus? Currently if a user clicks wildly around - i.e. between adress bar and content, this triggers each time a backend call. I would be nice to have a way to allow this i.e. only once per minute.

**Comment by phryneas** [comment]
You would have to use a custom handler for that or use a custom middleware that just throws away any `handleFocus` action if that action has been dispatched within the last X seconds.
